src/utils/document.js

```diff
@@ -100,6 +100,10 @@ export const getDocumentDatastoreAvailability = (document) => {
     return `${fileStatus === 'critical' && clean_end && !file_schema_valid ? 'Partial' : 'Yes'} - ${formatedDate}`;
   }
 
+  if (last_solrize_end) {
```

Comment:
See comment here: https://github.com/IATI/refresher/pull/278#discussion_r1311685101

I think refresher will need to unset solrize_end where data would normally be erased, so this function can check `solrize_end === null && last_solrize_end` to correctly identify Old versions.

Reply:
See other comment thread